src/app/feature/map/ui-controllers/floor-switcher/floor-switcher.component.ts

```diff
@@ -7,15 +7,25 @@ import { FloorService } from '../../../../core/services';
   styleUrls: ['./floor-switcher.component.scss'],
 })
 export class FloorSwitcherComponent {
-  constructor(private readonly _floorService: FloorService) {}
+  constructor(private readonly _floorService: FloorService) {
+    this.floorList = _floorService.floorList;
```

Comment:
Please check can we assign directly in property  `public readonly floorList  = this._floorService.floorList;`